benchmarks/nsinker/nsinker.cc

```diff
@@ -34,7 +34,10 @@ namespace aspect
          */
         NSinkerMaterial ();
 
-        double value (const Point<dim> &/*p*/) const;
+        /**
+         * Return the viscosity value at the given point `p`.
+         */
+        double value (const Point<dim> &p) const;
```

Comment:
I don't think this is correct. It returns the chi, which used to compute the viscosity.